tests/unit/s2n_tls13_cert_request_extensions_test.c

```diff
@@ -32,71 +32,7 @@ int main(int argc, char **argv)
     BEGIN_TEST();
     EXPECT_SUCCESS(s2n_enable_tls13());
 
-    /* Test client fails to parse certificate request with no extensions */
-    {
-        struct s2n_connection *client_conn;
-        EXPECT_NOT_NULL(client_conn = s2n_connection_new(S2N_CLIENT));
-        client_conn->actual_protocol_version = S2N_TLS13;
-
-        /* Write 0 length request context https://tools.ietf.org/html/rfc8446#section-4.3.2 */
-        EXPECT_SUCCESS(s2n_stuffer_write_uint8(&client_conn->handshake.io, 0));
-        /* write total extension length */
-        EXPECT_SUCCESS(s2n_stuffer_write_uint16(&client_conn->handshake.io, 0));
-
-        EXPECT_FAILURE_WITH_ERRNO(s2n_tls13_cert_req_recv(client_conn), S2N_ERR_MISSING_EXTENSION);
-
-        EXPECT_SUCCESS(s2n_connection_free(client_conn));
-    }
-
-    /* Test client fails to parse certificate request with wrong extension type */
```

Comment:
These tests are removed because they deal with sizes, and extension sizes are no longer handled manually?

Reply:
More than that, they were written to cover code that no longer exists. We used to have multiple copies of the extension parsing/processing code. These tests were for the cert_request version of that code. We now only have one copy of that code, and it is tested by the extension_list_* tests.